base_global_planner: NavfnROS
 
#base_local_planner: DWAPlannerROS
#base_local_planner: nav_core_adapter::LocalPlannerAdapter
#LocalPlannerAdapter:
  #planner_name: dwb_local_planner::DWBLocalPlanner

planners:
  - name: NavfnROS
    type: navfn/NavfnROS
NavfnROS:
  allow_unknown: false

controllers:
  - name: DWAPlannerROS
    type: dwa_local_planner/DWAPlannerROS
#  - name: DWBLocalPlanner 
#    type: nav_core_adapter::LocalPlannerAdapter

recovery_behaviors:
  - name: clear_costmap_recovery
    type: clear_costmap_recovery/ClearCostmapRecovery
clear_costmap_recovery:
  reset_distance: 0.0
  force_updating: true
  affected_maps: both
  layer_names: [obstacles1, obstacles2, obstacles3]
###########################################
shutdown_costmaps: false
controller_frequency: 20.0
planner_patience: 5.0
controller_patience: 15.0
conservative_reset_dist: 3.0
planner_frequency: 10.0
oscillation_timeout: 15.0
oscillation_distance: 0.5
robot_frame: $(arg mvibot_seri)/base_footprint
map_frame: map

